app/controllers/texts_controller.rb

```diff
@@ -2,4 +2,8 @@ class TextsController < ApplicationController
   def index
     @texts = Text.all
   end
+
+  def show
+    @text = Text.find_by(id: params[:id])
```

Comment:
こういう場合は `find` を使うべきです。`find_by` では存在しない場合でもエラーが出ません。

```suggestion
    @text = Text.find(params[:id])
```